sources/host-ctr/go.mod

```diff
@@ -16,16 +16,6 @@ require (
 )
 
 replace (
```

Comment:
You may want to run `go mod tidy`  to get things cleaned up since we don't need the multiline replace anymore. Go should re-format it for you as:

```
// containerd still uses the v1alpha2 APIs in k8s.io/cri-api.
// These were removed in v0.26 of cri-api resulting in a dependency conflict.
// TODO: Remove this `replace` when containerd uses v1 of the cri-api
replace k8s.io/cri-api => k8s.io/cri-api v0.25.5
```